bottlerocket-settings-models/settings-models/Cargo.toml

```diff
@@ -29,6 +29,7 @@ settings-extension-bootstrap-containers.workspace = true
 settings-extension-cloudformation.workspace = true
 settings-extension-container-registry.workspace = true
 settings-extension-container-runtime.workspace = true
+settings-extension-container-runtime-plugins.workspace = true
```

Comment:
We will need to bump the version of `settings-models` and `modeled-types` prior to releasing. Might as well add it in this PR.